src/components/api-logic.js

```diff
@@ -33,3 +33,19 @@ export const FetchUsers = (setIsLoading, setUsers) => {
       });
   }, [setIsLoading, setUsers]);
 };
+
+export const FetchSingleArticle = (
```

Comment:
These should just be functions, not inside of useEffects. The useEffect hook happens in the component, where we would invoke the api function.